src/main/java/net/canfar/storage/web/resources/StorageItemServerResource.java

```diff
@@ -473,6 +476,15 @@ <T> T executeSecurely(final Subject subject, final PrivilegedExceptionAction<T>
         }
     }
 
+    String getDisplayName() throws Exception {
+        final IdentityManager identityManager = AuthenticationUtil.getIdentityManager();
+        return identityManager.toDisplayString(getVOSpaceCallingSubject());
+    }
+
+    Subject getVOSpaceCallingSubject() throws Exception {
```

Comment:
Are you on-purpose over-riding the method `getCallingSubject`, without doing language-supported over-riding? If so, perhaps add a comment to say why this is necessary?